host/src/lib/HelperFunctions.jsx

```diff
@@ -291,6 +337,15 @@ export const checkEqualityWithOtherAnswers = (rawAnswer, normValue, normType, pr
   return false;
 }
 
+/**
+ * This function creates the short answer responses object that will be stored in the question object, via equality checks
+ * @param {IResponse} prevShortAnswer 
+ * @param {IChoice[]} choices 
+ * @param {any}} newAnswer 
+ * @param {any} newAnswerTeamName 
```

Comment:
Extra `{` on line 344. Are these really of type `any`? Can we have them to have a strong type?